common/utils/utils.go

```diff
@@ -349,20 +349,24 @@ func ParseNormalizedVolumeID(ctx context.Context, volID string) (string, int, st
 		return "", 0, "", "", fmt.Errorf("volume ID '%s' cannot be split into tokens", volID)
 	}
 
-	var clusterName string
+	volumeName := tokens[0]
+
 	exportID, err := strconv.Atoi(tokens[1])
 	if err != nil {
 		return "", 0, "", "", err
 	}
 
+	accessZone := tokens[2]
```

Comment:
Please update the description specifying the need for the changes in this file.

Reply:
Done